### PR TITLE
CNF-846 - Deploy a VRF CNI plugin that makes it possible to assign different secondary networks to different VRF - release note

### PR DESCRIPTION
**Pull request by aireilly**
Release Note for https://issues.redhat.com/browse/CNF-846

### CODE REVIEW

**Comment by aireilly** [comment]
@fedepaol @kononovn release note for CNF-847 OCP 4.7 release. Please review :)

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -120,6 +120,12 @@ The xref:../support/remote_health_monitoring/using-insights-to-identify-issues-w
 [id="ocp-4-7-scale"]
 === Scale
 
+[id="ocp-4-7-scale-new-vrf-cni-plugin"]
+==== New VRF CNI plugin allows secondary networks to be assigned to different VRFs
+
+A new VRF CNI plugin is now available which allows you to assign additional networks to network interfaces. When you specify an additional network using a `rawCNIConfig` configuration for the CNO custom resource, the CNO creates the `NetworkAttachmentDefinition` custom resource automatically.
```
Comment (fedepaol) [inline release_notes/ocp-4-7-release-notes.adoc:126]: I'd reword this in a way where we associate additional networks (which we are already able to create) to a vrf using the extra configuration.
In this way, it looks like the VRF plugin is the one allowing the additional networks, whereas it just associates them to a VRF.

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -302,6 +302,16 @@ The latency test, a part of the CNF-test container, provides a way to measure if
 
 For information about running a latency test, see xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#cnf-performing-end-to-end-tests-running-the-latency_tests[Running the latency tests].
 
+[id="ocp-4-7-scale-new-vrf-cni-plugin"]
+==== New VRF CNI plugin allows secondary networks to be assigned to VRFs
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:306]: plug-in

File: release_notes/ocp-4-7-release-notes.adoc [hunk 3]
```diff
@@ -302,6 +302,16 @@ The latency test, a part of the CNF-test container, provides a way to measure if
...
+
+A new VRF CNI plugin is now available which allows you to assign additional networks to a VRF. When you create a secondary network using a `rawConfig` configuration for the CNO custom resource and configure a VRF for it, the interface created for the pod is associated with the VRF. You can also use the VRF CNI plug-in to assign an SR-IOV network to a VRF.
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:308]: plug-in is now available, which

File: release_notes/ocp-4-7-release-notes.adoc [hunk 4]
```diff
@@ -302,6 +302,16 @@ The latency test, a part of the CNF-test container, provides a way to measure if
...
+
+For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF] and xref:../networking/hardware_networks/configuring-sriov-device.adoc#cnf-assigning-a-sriov-network-to-a-vrf_configuring-sriov-device[Assigning an SR-IOV network to a VRF].
+
+
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:312]: A few unnecessary extra lines here

**Comment by ahardin-rh** [comment]
@aireilly Just some minor nits from me. Looks good!

File: release_notes/ocp-4-7-release-notes.adoc (hunk 3 above)
Comment (aireilly) [reply release_notes/ocp-4-7-release-notes.adoc:308]: hmm. I have a small quibble with this. I don't think the first clause takes a comma. 

All I can find in the IBM style guide is the following: "Use which, preceded by a comma, to introduce a nonrestrictive clause. A nonrestrictive clause provides extra information that is not essential to the meaning of the sentence. If the nonrestrictive clause is removed, the basic meaning of the sentence does not change."

This is not a nonrestrictive clause...

Comment (ahardin-rh) [reply release_notes/ocp-4-7-release-notes.adoc]: Ah, okay. My hypercorrection 🙂. Sorry! As long as plugin is updated to plug-in we should be good to go!
